.vsts.pipelines/builds/matrix.yml

```diff
@@ -96,4 +96,4 @@ jobs:
         artifactName: Tarball debian9 Online
       - job: fedora30
       - job: fedora31
-      gatherPortableJob: centos71
+      gatherPortableJob: fedora30
```

Comment:
Any particular reason for picking 30? It's been [EOL'ed for a while now](https://lists.fedoraproject.org/archives/list/devel@lists.fedoraproject.org/message/7UTUFY7WEL6RTFRXJB75XAFH44Y6RPUC/).

Reply:
It's the one we're building Portable.  I'd rather not delay this PR any further, but this should be updated.  I filed an issue here: https://github.com/dotnet/source-build/issues/1780